### PR TITLE
reduced font size of Facebook btn

### PR DESCRIPTION
**Pull request by jessevdp**
Text inside Facebook button was still displaying weirdly on some pages.

Reducing its font size (again) fixes it.

Fixes issue #5 (again)

### CODE REVIEW

**Comment by lesander** [comment]
This is a rather dirty hack. I'd rather have a different `em` size for each viewport.

**Comment by jessevdp** [comment]
That's not how it used to be but probably a best practise. I'll look into it.

**Comment by jessevdp** [comment]
Think it's all ready to merge now. Take a look at the `font-size` yourself to see if you want to make any changes.

File: _sass/scale.scss [hunk 1]
```diff
@@ -110,3 +110,22 @@ nav {
...
   }
 }
+
+/* Facebook button (news page & contact page) */
+.btn-fb {
+  font-size: 0.9rem;
+
+  /* responsive displays */
+  @media #{$mq-max-xsmall} {
+    font-size: 1rem !important;
```
Comment (lesander) [inline _sass/scale.scss]: Are all those `!important`s required? If I'm correct, just the media queries will override the `font-size`.

**Review by lesander — CHANGES_REQUESTED** [review]

File: _sass/scale.scss (hunk 1 above)
Comment (jessevdp) [reply _sass/scale.scss]: I suppose they're not needed anymore, while working on it I hadn't deleted the old styles yet... But I guess a simple test should tell us the answer...

**Comment by jessevdp** [comment]
Ah, the media query **is being overwritten** as you can see in the following screenshot. I'll delete the css that overwrites it as well as the `!important` tags.

(this only happens on the news page)

screen shot 2017-01-21 at 14 51 14

File: _sass/scale.scss (hunk 1 above)
Comment (jessevdp) [reply _sass/scale.scss]: Ah, the media query **is being overwritten** as you can see in the following screenshot. I'll delete the css that overwrites it as well as the `!important` tags.

(this only happens on the news page)

screen shot 2017-01-21 at 14 51 14

Comment (jessevdp) [reply _sass/scale.scss]: Wait a minute, can't really delete the css since it does not ***only*** affect the fb button but all links on the page.

Comment (lesander) [reply _sass/scale.scss]: I suppose you could create a new css selector specifically for the affected buttons.

**Review by lesander — APPROVED** [review]

**Pull request merged**